src/helpers/elements-interactions.js

```diff
@@ -11,8 +11,7 @@ async function fillInput(page, inputSelector, inputValue) {
 }
 
 async function clickButton(page, buttonSelector) {
-  const button = await page.$(buttonSelector);
-  await button.click();
+  await page.$eval(buttonSelector, el => el.click());
```

Comment:
@baruchiro I wonder if we need to change it for all button clicks
I understand it is required in this use case, I would probably create a new function under `elements-interactions` for clicking an `a` element

Reply:
Okay, let's just start just in case of `a`.